UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Controllers/ProductDataControllerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using FakeItEasy;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.Controllers;
+using UKHO.ExchangeSetService.API.Services;
+using UKHO.ExchangeSetService.Common.Models.Request;
+using UKHO.ExchangeSetService.Common.Models.Response;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Controllers
```

Comment:
Are these unit tests... or component tests? Unit tests test functionality within a single class

Reply:
As far as i can see the only class being tested is the controller? can you elaborate on what you mean?

Reply:
Yes my bad.